docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -38,6 +38,8 @@ GROUP BY GROUPING SETS (grouping_set [ , ...])
 While aggregate functions are defined as
 ```sql
 aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_expression ) ]
+
+[ FIRST | LAST ] ( expression [ IGNORE NULLS ] ) [ FILTER ( WHERE boolean_expression ) ]
```

Comment:
hm, this syntax definition only for `FIRST/LAST` looks verbose... can we remove it then describe something about `FIRST/LAST` in the `**aggreagate_name**` sub-section? Actually, I think `FIRST/LAST` can be used together with `DISTINCT`; https://github.com/apache/spark/pull/29143